visualisation/nzvm.py

```diff
@@ -129,11 +129,12 @@ def main():
         print("Error: Inconsistent VM size. Check model and nx, ny, nz parameters and try again. Expected",nx*ny*nz,"values, got",len(array))
         quit()
 
-    #yzx
+    km_to_m_multiplier = 1000
```

Comment:
Constants are generally defined after the imports and in all caps. (they aren't strictly constants in python, but globals that behave like constants in other languages)